hedy.py

```diff
@@ -2830,6 +2814,168 @@ def print_empty_brackets(self, meta, args):
         return self.print(meta, args)
 
 
+@v_args(meta=True)
```

Comment:
After some thought and a brief micro:bit research, I think it will be best to have a separate stack of transpilers for microbit. The main motivation is that most commands will be transpiled in the same way, but those that need to be changed would be very different from the original code. For example, micro:bit does not have support for f-strings (right?), which means that the print and ask commands would need to be handled differently.

I already set up a framework for the new transpilers. Every microbit transpiler class for level N will inherit from the microbit transpiler of level N-1 and the original transpiler for level N. In this way, the new code would need to be minimal.

Reply:
Thank you for pointing out this approach. As we discussed and agreed in our meeting I'll use it and continue with it. Thank you once again for writing the skeleton for the microbit transpiler. I'll continue to add more functionality to it.